src/store/modules/transferorder/actions.ts

```diff
@@ -18,6 +18,7 @@ const actions: ActionTree<TransferOrderState, RootState> = {
     emitter.emit('presentLoader');
     let resp;
     const transferOrderQuery = JSON.parse(JSON.stringify(state.transferOrder.query))
+    const currentFacilityId = getCurrentFacilityId();
```

Comment:
Use the getCurrentFacilityId() directly if currentFacilityId variable is not reused.